Neos.Flow/composer.json

```diff
@@ -32,6 +31,7 @@
         "doctrine/orm": "~2.5.0",
         "doctrine/migrations": "~1.3.0",
         "doctrine/dbal": "~2.5.0",
+        "doctrine/common": ">=2.4,<2.8-dev",
```

Comment:
In fact this is copied from the `doctrine/dbal` and `doctrine/orm` requirements as I thought it makes sense to match it to them...